tests/testthat/resources/file exists.Rmd

```diff
@@ -0,0 +1,6 @@
+
+```{r}
+if (!file.exists("empty.csv"))
+   stop("empty.csv expected in ", getwd())
+```
+
```

Comment:
This filename will trigger a warning in `R CMD check`. I'm surprised that we missed it because we didn't enable Travis CI for this repo: https://travis-ci.org/profile/rstudio Can someone with admin privilege enable it? @jjallaire